commit 5eb784f4a34535dc74a10336b04393fafe554fab
Author: Maciej S. Szmigiero <mail@maciej.szmigiero.name>
Date:   Sat Jan 21 23:50:18 2017 +0100

    w1: don't leak refcount on slave attach failure in w1_attach_slave_device()
    
    commit d2ce4ea1a0b0162e5d2d7e7942ab6f5cc2063d5a upstream.
    
    Near the beginning of w1_attach_slave_device() we increment a w1 master
    reference count.
    Later, when we are going to exit this function without actually attaching
    a slave device (due to failure of __w1_attach_slave_device()) we need to
    decrement this reference count back.
    
    Signed-off-by: Maciej S. Szmigiero <mail@maciej.szmigiero.name>
    Fixes: 9fcbbac5ded489 ("w1: process w1 netlink commands in w1_process thread")
    Cc: Evgeniy Polyakov <zbr@ioremap.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1306_5eb784f4a34535dc74a10336b04393fafe554fab.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1306_5eb784f4a34535dc74a10336b04393fafe554fab.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
